src/Components/SystemDetail/RecommendationsTable.js

```diff
@@ -12,10 +12,12 @@ const renderExpandedView = (row) => {
     return (
         <TextContent className='ros-rec-content'>
             <Text component={TextVariants.p}>
+                <Text style={{ fontWeight: 'bold' }}>Detected issues</Text>
                 {row.reason}
             </Text>
```

Comment:
Attached screenshot vs mockup [page](https://marvelapp.com/prototype/geig8bh/screen/80186730):
- Blank space between two paragraphs is missing.